### DISCUSSION TITLE
Airplay Support

### DISCUSSION THREAD

**Discussion by mirisbowring** [advplyr/audiobookshelf-app#751] [Ideas]
Wanted to ask whether this great app is going to support Airplay or another cast function?

We have a central Multizone Audio at home and bluetooth is too far away and we don't want to continuously repair.

**Comment by advplyr** [comment]
A request was opened for this here https://github.com/advplyr/audiobookshelf-app/issues/779

This is supported but there are just no buttons in the app itself

**Reply by mirisbowring** [reply]
Ah, thanks for the hint - i am relatively new to the ios world.

**Comment by jt196** [comment]
Hey so I've got some Airplay speakers at home (Sonos) and while the Android app supports casting to Android devices, it doesn't support airplay devices. Would it be possible to just cast to Airplay devices as well as the android ones using the same button?
